subgraphs/makerdao/src/common/helpers.ts

```diff
@@ -510,10 +509,18 @@ export function updatePosition(
         PositionSide.LENDER,
         true
       );
-      assert(
-        deltaCollateral.gt(BIGINT_ZERO),
-        `[updatePosition]Creating new lender position ${lenderPosition.id} with deltaCollateral ${deltaCollateral} <= 0`
-      );
+
+      if (deltaCollateral.le(BIGINT_ZERO)) {
+        log.critical(
```

Comment:
From what I've seen, `log.criticals` don't output the message to the logs. Instead do a `log.error` with all the data you want to log, followed by a `log.critical` to then stop the execution

Reply:
Fixed.